java/HealthVault Mobile/HVSample/build.gradle

```diff
@@ -9,7 +9,11 @@ android {
         minSdkVersion 23
         targetSdkVersion 23
     }
-
+    repositories {
```

Comment:
Not needed

---
In reply to: [118754382](https://github.com/Microsoft/healthvault-samples/pull/13#discussion_r118754382) [](ancestors = 118754382)